src/app/screens/LNURLPay/LNURLPayForm.tsx

```diff
@@ -43,21 +39,12 @@ const Dd = ({ children }: { children: React.ReactNode }) => (
   <dd className="mb-4 text-gray-600 dark:text-neutral-500">{children}</dd>
 );
 
-function LNURLPay(props: Props) {
+const LNURLPayForm: FC<Props> = ({ origin, details, isPrompt }) => {
```

Comment:
what is FC?

Reply:
Correct, just a differrent way of writing a Functional Component.

`function MyComponent(props: Props) {....`
`const MyComponent: FC<Props> = (props) => {....`